Source/Core/DolphinQt/SkylanderPortal/SkylanderModifyDialog.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include <QDialog>
+#include <QCheckBox>
```

Comment:
Instead of an `#include` you can just forward declare `QCheckBox`, like what's done for `QVBoxLayout` and `QDialogButtonBox` at around line 12.